src/urllib3/filepost.py

```diff
@@ -54,6 +54,7 @@ def encode_multipart_formdata(
 
     :param fields:
         Dictionary of fields or list of (key, :class:`~urllib3.fields.RequestField`).
+        Values are processed by :func:`urllib3.fields.RequestField.from_tuples`.
```

Comment:
Not sure if you need the code because [`from_tuples()`](https://urllib3--2505.org.readthedocs.build/en/2505/reference/urllib3.fields.html#urllib3.fields.RequestField.from_tuples) has examples. I think my journey went "i can't work out this dict of fields, read source, go to `from_tuples()`, ah the examples I need to grok this!".

The reason the history is a bit foggy is that actually I thought I had to create `RequestField`s myself, call `make_multipart()` on each one, and then pass a list of tuples `(key, request_field)` to `encode_multipart_formdata()`. That is what I ended up doing a few months ago. Only two days ago when reading that code to re-use it somewhere else, with a mixture of files and plain parameters, did I read the docs closely enough to realise there was an easier way.